samples/sas_resource_files/sas_resources_files_example.R

```diff
@@ -1,30 +1,45 @@
 library(doAzureParallel)
 
 doAzureParallel::setCredentials("credentials.json")
-storageAccountName <- "<YOUR_STORAGE_ACCOUNT>"
+# Using rAzureBatch directly for storage uploads
+config <- rjson::fromJSON(file = paste0("credentials.json"))
+
+storageCredentials <- rAzureBatch::SharedKeyCredentials$new(
+  name = config$storageAccount$name,
+  key = config$storageAccount$key
+)
+
+storageAccountName <- storageCredentials$name
 inputContainerName <- "datasets"
 
-# Generate a sas tokens with the createSasToken function
+storageClient <- rAzureBatch::StorageServiceClient$new(
+  authentication = storageCredentials,
+  url = sprintf("https://%s.blob.%s",
+               storageCredentials$name,
+               "core.windows.net"
```

Comment:
May be better to use endpoint from config file so the sample can run in national clouds as well.